src/Microsoft.Azure.SignalR.AspNet/ServiceOptions.cs

```diff
@@ -72,6 +72,8 @@ public class ServiceOptions : IServiceEndpointOptions
         /// </summary>
         public int? DisconnectTimeoutInSeconds { get; set; }
 
+        public AzureAdOptions AzureAdOptions { get; set; } = new AzureAdOptions();

```

Comment:
Add an auth option and move aad options inside, in case we can support more auth type later?

Reply:
make it internal first